Voice SRTP Support (#3636)
Nexmo/nexmo-developer: _documentation/en/voice/voice-api/ncco-reference.md

Value | Description
-- | --
`type` | The endpoint type: `sip` for SIP.

> `uri` | the SIP URI to the endpoint you are connecting to in the format `sip:rebekka@sip.example.com`. To have [TLS and/or SRTP](/voice/sip/overview#protocols) being used, include respectively `transport=tls` or `media=srtp` to the URL with the semicolon `;` as a delimiter, for example: `sip:rebekka@sip.example.com;transport=tls;media=srtp`. 

Reviewer: ```suggestion
`uri` | the SIP URI to the endpoint you are connecting to in the format `sip:rebekka@sip.example.com`. To use [TLS and/or SRTP](/voice/sip/overview#protocols), include respectively `transport=tls` or `media=srtp` to the URL with the semicolon `;` as a delimiter, for example: `sip:rebekka@sip.example.com;transport=tls;media=srtp`. 
```
